Feature/currency cb rf(oophion n)

OophionN opened the pull request:
rename branch

Maksar503 committed:
Добавить скрипт выгрузки курса валют и ReadMe

OophionN committed:
Update ReadMe.md

OophionN committed:
update parser

Добавить вызов utils скорректировать относительные пути для сохранения результатов

OophionN committed:
Delete currency_CB.py

Удалить в связи с загрузкой обновленной версии - актуальный файл parser_CB_ver3

PhySci commented on script_currency_CB_RF/parser_CB_ver3.py:
```diff
@@ -0,0 +1,78 @@
+def save_to_csv(data, file_path):
+    try:
+        now = datetime.now().strftime("%Y-%m-%d")
+        file_path_with_date = os.path.join('data', f"{file_path}_{now}.csv")
```
Отлично! Здорово что используешь библиотеку os.path

OophionN replied:
Спасибо. Открыл ее для себя когда отрабатывал рекомендации Юли, раньше не пользовался. Добавил в подборку на будущее, как понял пока она мне полезна в скриптах.

PhySci commented on script_currency_CB_RF/parser_CB_ver3.py:
```diff
@@ -0,0 +1,78 @@
+
+def save_to_csv(data, file_path):
+    try:
+        now = datetime.now().strftime("%Y-%m-%d")
```
Эту и следующую строчку лучше вынести из блока try-except. Ведь в них нечему ломаться.

OophionN replied:
Сделано

PhySci commented on script_currency_CB_RF/parser_CB_ver3.py:
```diff
@@ -0,0 +1,78 @@
+
+if __name__ == "__main__":
+    setup_logging('parser.log')
+    import logging
```
Подними этот импорт на первые строчки, там же где и другие.

OophionN replied:
забыл перенести пока тестил. Сделано

PhySci reviewed (changes requested):
Переименуй, пожалуйста, файл так чтобы в нем не было ver3. Почему бы не назвать его _parser_currency.py_ Сокращение CB не самое очевидное. 
 И перенеси в src/parser как обсуждали ранее.

OophionN committed:
Add files via upload

учтены замечания из Feature/currency cb rf(oophion n) #19 (корректировка названия, оптимизация try/except, перенос в ветку scr/parser скрипта)

OophionN committed:
Delete parser_CB_ver3.py

удаление в связи с переносом скрипта в директорию scr/parser

OophionN commented:
Все замечания отработал, файл в старой директории удалил, саму директорию оставил, так как там есть ридми

PhySci commented:
> саму директорию оставил, так как там есть ридми

Директория ради одного файла readme выглядит не очень. Не очень понятно, почему эта папка должна существовать.
 Давай самую важную информацию перенесём в заголовок скрипта, а файл и папку удалим.

OophionN commented:
@PhySci Федор, я удалил директорию с README. Общая конструкция ветки соответствует main.
